### ISSUE TITLE
Higher batch sizes consume more memory (batch size vs batch count)

### ISSUE THREAD

**Issue by bastim94** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#226]
The more images you want to process the higher your GPU memory consumption goes up which is a problem if you want 20 variants at once.
Seems like the plugin wants to process all in one go which sounds it uses for the amount of images the "batch size" feature instead of doing them in a queue like you can do with with "batch count" in stable diffusion.

Is there a possibilty to get two different input fields so we can chosse at our own which one to use?

**Comment by AbdullahAlfaraj** [comment]
Yes, the plugin now supports both batch size and batch count, see: #221
Though the feature hasn't been packaged and released yet.

**Comment by eienatsu** [comment]
> Yes, the plugin now supports both batch size and batch count, see: #221 Though the feature hasn't been packaged and released yet.

Will this feature be packed in a new .ccx file? The latest version v1.2.2 does not appear to have this feature yet.

P.S. Awesome plugin!
